examples/hyperparam/conda.yaml

```diff
@@ -1,20 +1,20 @@
 name: hyperparam_example
 channels:
-  - defaults
-  - anaconda
   - conda-forge
 dependencies:
   - python=3.6
-  - numpy=1.14.3
-  - pandas=0.22.0
-  - scikit-learn=0.19.1
-  - matplotlib=2.2.2
-  - keras==2.2.2
   - pip
   - pip:
-    - mlflow>=1.0
-    - Gpy==1.9.2
+    - numpy==1.19.5
+    - click
+    - pandas==0.22.0
+    - scipy==1.4.1
```

Comment:
I fix this version to 1.4.1 because higher version will cause test test_mlflow_run_example[hyperparam-params3 failure:

```
packages/scipy/optimize/_numdiff.py", line 388, in approx_derivative
    raise ValueError("`f0` passed has more than 1 dimension.")
ValueError: `f0` passed has more than 1 dimension.
```